guides/queries/multiplex.md

```diff
@@ -78,12 +78,10 @@ def execute
     )
   end
 
-  render json: result
+  render json: result, root: false
 end
 ```
 
-If Apollo Client has issues recognizing the result of `render json: result`, replace it with `render body: result.to_json, content_type: 'application/json'`.
```

Comment:
Did some looking around and found https://github.com/rmosolgo/graphql-ruby/issues/1948 and https://github.com/rmosolgo/graphql-ruby/pull/1949 for the reasoning behind this additional option. They were having the same issue of `render json: result` returning { 'graphql': result } 😃